src/server/verification/verification.js

```diff
@@ -92,7 +92,7 @@ class Verifications implements VerificationAPI {
    * @param {string} verificationData.code - code used to verify that the email is valid
    * @returns {Promise<boolean|Error>}
    */
-  async verifyEmail(user: UserRecord, verificationData: { code: string }): Promise<boolean | Error> {
+  async verifyEmail(user: UserRecord, verificationData: { code: integer }): Promise<boolean | Error> {
```

Comment:
just a _type_ related thing.

Despite is composed of numbers, the code is stored as a string

https://github.com/GoodDollar/GoodServer/blob/3ba50f600e1345a4b4e72dfb83a268a8f9136894/src/imports/otp.js#L17

```suggestion
  async verifyEmail(user: UserRecord, verificationData: { code: string }): Promise<boolean | Error> {
```